pkg/io/video/convert.go

```diff
@@ -3,8 +3,38 @@ package video
 import (
 	"fmt"
 	"image"
+	"image/color"
 )
 
+// imageToYCbCr converts image to *image.YCbCr
+// Note: conversion can be lossy
+func imageToYCbCr(img image.Image) *image.YCbCr {
+	yuvImg, ok := img.(*image.YCbCr)
+	if ok {
+		return yuvImg
+	}
+
+	bounds := img.Bounds()
+	yuvImg = image.NewYCbCr(bounds, image.YCbCrSubsampleRatio444)
+	dy := bounds.Max.Y - bounds.Min.Y
+	dx := bounds.Max.X - bounds.Min.X
+	i := 0
+	for yi := 0; yi < dy; yi++ {
+		for xi := 0; xi < dx; xi++ {
+			// TODO: probably try to get the alpha value with something like
+			// https://en.wikipedia.org/wiki/Alpha_compositing
+			r, g, b, _ := img.At(xi, yi).RGBA()
+			yy, cb, cr := color.RGBToYCbCr(uint8(r/256), uint8(g/256), uint8(b/256))
+			yuvImg.Y[i] = yy
+			yuvImg.Cb[i] = cb
+			yuvImg.Cr[i] = cr
+			i++
```

Comment:
Input image can be padded.
It's safer to do like
https://github.com/pion/mediadevices/blob/cad9f410477aa2611c18940d54336a56121272d8/pkg/driver/screen/x11_linux.go#L77-L90
(This block can be removed in this PR!)

Reply:
hmm, not sure what you mean by padded. Do you mean about `YStride` and `CStride`?

Reply:
Oh, sorry. This image is allocated by us, so we don't have to take padding of the image into account.
(I meant for example `Width=15, YStride=16 (4-byte aligned, 1-byte padded), Rect.Max.X = 15`)

Reply:
Yep, we have control over this. And, to make it simple, we'll just always convert it to I444.